docs/specification/1.0.md

```diff
@@ -736,9 +736,9 @@ Field | Optionality | Prefetch Token | Type | Description
 
 #### FHIR resources in context
 
-When potentially multiple FHIR resources value a single context field, these resources SHOULD be formatted as a FHIR Bundle. For example, multiple FHIR resources are necessary to describe all of the orders under review in the `order-review` hook's `orders` field. Hook definitions SHOULD prefer the use of FHIR Bundles over other bespoke data structures.
+When a context field may contain multiple FHIR resources, these resources SHOULD be formatted as a FHIR Bundle. For example, multiple FHIR resources are necessary to describe all of the orders under review in the `order-review` hook's `orders` field. Hook definitions SHOULD prefer the use of FHIR Bundles over other bespoke data structures.
```

Comment:
a bit awkward -- When a context field contains multiple FHIR resources

Reply:
How about "For context fields that may contain multiple FHIR resources, the field SHOULD be defined as a FHIR Bundle, rather than as an array of FHIR resources."?